src/Symfony/Component/ObjectMapper/ObjectMapper.php

```diff
@@ -319,4 +321,13 @@ private function getSourceReflectionClass(object $source, \ReflectionClass $targ
 
         return $targetRefl;
     }
+
+    #[Required]
```

Comment:
I'm wondering if this is allowed or if I should add something to the framework bundle, I could not find any other occurence of something like this in the current code base.

Reply:
this should be removed IMHO: this attribute marks a method that should be called just after the constructor
but here, the constructor already accepts a mapper, so that calling this method is not needed to have the object fully instantiated